### PR TITLE
Cleanup the searchpage and add unit tests for user entity

### PR DESCRIPTION
**Pull request by Zhuofan-Y**
Clean up the comments and fixed warnings and typos. Also adds a new function that it will clear the input text field when user hits submit button. Comment out the open fridge button since the usecase for that is not implemented.

For unit test for user entity:
Tested:{
1. get name function for CommonUser
2. get fridge function for CommonUser
3. create User function for CommonUserFactory
}

### CODE REVIEW

**Review by EricKarpovits — APPROVED** [review]
Good small fixes!

File: src/main/java/UI/searchPageHelpFrame.java [hunk 1]
```diff
@@ -64,15 +60,15 @@ public searchPageHelpFrame(UserEnterIngredientsController ingredientsController,
         panel1.setLayout(null);
 
         //create open_fridge button which pops up a messagebox of what ingredients fridge has.
-        open_fridge = new JButton("open your fridge");
-        open_fridge.setBounds(300, 80, 240, 80);
-        open_fridge.addActionListener(this);
+//        open_fridge = new JButton("open your fridge");
+//        open_fridge.setBounds(300, 80, 240, 80);
+//        open_fridge.addActionListener(this);
```
Comment (EricKarpovits) [inline src/main/java/UI/searchPageHelpFrame.java]: Delete the open_fridge code we will no longer use it.

File: src/main/java/UI/searchPageHelpFrame.java [hunk 2]
```diff
@@ -64,15 +60,15 @@ public searchPageHelpFrame(UserEnterIngredientsController ingredientsController,
...
 
         //sets up panel2 and add open_fridge button to it
         JPanel panel2 = new JPanel();
         panel2.setBounds(0, 240, 840, 480);
         panel2.setLayout(null);
-        panel2.add(open_fridge);
+//        panel2.add(open_fridge);
```
Comment (EricKarpovits) [inline src/main/java/UI/searchPageHelpFrame.java:67]: Delete any commented out code

**Review by EricKarpovits — CHANGES_REQUESTED** [review]

File: src/main/java/UI/searchPageHelpFrame.java (hunk 2 above)
Comment (Zhuofan-Y) [reply src/main/java/UI/searchPageHelpFrame.java:67]: Done!

File: src/main/java/UI/searchPageHelpFrame.java (hunk 1 above)
Comment (Zhuofan-Y) [reply src/main/java/UI/searchPageHelpFrame.java]: Deleted

**Review by EricKarpovits — APPROVED** [review]
Great work @Zhuofan-Y

**Review by RedFoxity — APPROVED** [review]
Looks good!

File: src/test/java/entities/user/CommonUserTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    void test_get_fridge(){
+        CommonIngredientFactory factory = new CommonIngredientFactory();
+        Ingredient chicken = factory.create("chicken");
+        ArrayList<CommonIngredient> ing = new ArrayList<>();
+        ing.add((CommonIngredient) chicken);
+        CommonFridge fridge = new CommonFridge(ing);
+        CommonUser user = new CommonUser("D.Jr", fridge);
+        Assertions.assertEquals(fridge, user.getFridge());
+    }
+}
```
Comment (fishyvishy) [inline src/test/java/entities/user/CommonUserTest.java:45]: could add tests for empty fridge case as well.

**Review by fishyvishy — APPROVED** [review]
Looks good. If necessary, could add empty case tests.
